accio-base/src/main/java/io/accio/base/type/TimestampWithTimeZoneType.java

```diff
@@ -88,12 +89,25 @@ public Object decodeUTF8Text(byte[] bytes)
     @VisibleForTesting
     ZonedDateTime tryParse(String timeString)
     {
-        return ZonedDateTime.parse(timeString, PG_TIMESTAMP);
+        try {
+            // Postgres TimestampTz format
+            return ZonedDateTime.parse(timeString, PG_TIMESTAMP);
+        }
+        catch (Exception e) {
+            // JDBC transform ZonedDateTime to String
+            return ZonedDateTime.parse(timeString, ISO_ZONED_DATE_TIME);
+        }
```

Comment:
Pardon, this makes me a little bit confuse, why do we need to handle two different formats here ?

Reply:
oh, it's my mistake. I should use `PG_TIMESTAMP` to format `ZoneDateTime` in the test. We don't need to handle `ISO_ZONED_DATE_TIME` format.